doc/source/user/select-openstack-services.rst

```diff
@@ -1,10 +1,26 @@
-================================
-Select OpenStack services to run
-================================
+====================================
+Select OpenStack services to install
+====================================
 
-SUSE Containerized OpenStack currently deploys OpenStack Cinder,
-Glance, Heat, Horizon, Keystone, Neutron, Nova.
+SUSE Containerized OpenStack currently deploys OpenStack Cinder, Glance, Nova,
+Neutron, Heat, Horizon, and Keystone.
 
-You can decide which services to deploy by:
+You can change which services to deploy by modifying the chart group list in
+the site manifest file site/soc/software/manifests/full-site.yaml.
 
-TODO.
+.. code-block:: yaml
+
+   chart_groups:
+     - openstack-ingress-controller-soc
+     - openstack-mariadb-soc
+     - openstack-memcached-soc
+     - openstack-keystone-soc
+     - openstack-ceph-config-soc
+     - openstack-glance-soc
+     - openstack-cinder-soc
+     - openstack-compute-kit-soc
+     - openstack-heat-soc
+     - openstack-horizon-soc
```

Comment:
Hm, the file actually also mentions `openstack-tempest-soc` and deploys it conditionally